website/showcase.json

```diff
@@ -0,0 +1,514 @@
+    "infoTitle": "Why React Native worked well for us",
+    "pinned": false
+  },
+  {
+    "name": "Start",
+    "icon": "http://a1.mzstatic.com/us/r30/Purple49/v4/de/9b/6f/de9b6fe8-84ea-7a12-ba2c-0a6d6c7b10b0/icon175x175.png",
+    "linkAppStore": "https://itunes.apple.com/us/app/start-medication-manager-for/id1012099928?mt=8",
+    "infoLink": "http://www.nytimes.com/2014/09/24/technology/to-gather-drug-information-a-health-start-up-turns-to-consumers.html?_r=0",
+    "infoTitle": "NYT: A Health Startup Turns to Consumers",
+    "pinned": false
+  },
+  {
+    "name": "Taxfyle",
+    "icon": "https://s3.amazonaws.com/taxfyle-public/images/taxfyle-icon-1024px.png",
+    "linkAppStore": "https://itunes.apple.com/us/app/taxfyle/id1058033104?mt=8",
+    "infoLink": "http://www.techinsider.io/taxfyle-wants-to-be-the-uber-for-taxes-2016-4",
+    "infoTitle": "Taxfyle: the Uber for filing taxes",
+    "pinned": false
+  },
+  {
+    "name": "This AM",
+    "icon": "http://s3.r29static.com//bin/public/efe/x/1542038/image.png",
+    "linkAppStore": "https://itunes.apple.com/us/app/refinery29-this-am-top-breaking/id988472315?mt=8",
+    "infoLink": "https://techcrunch.com/2016/02/01/refinery29-debuts-its-first-app-a-morning-news-round-up-called-refinery29-am/",
+    "infoTitle": "Refinery29 debuts morning news roundup app created with React Native",
+    "pinned": false
+  },
+  {
+    "name": "TRED",
+    "icon": "http://a1.mzstatic.com/us/r30/Purple20/v4/b0/0c/07/b00c07d2-a057-06bc-6044-9fdab97f370f/icon175x175.jpeg",
+    "linkAppStore":  "https://itunes.apple.com/us/app/tred-sell-my-car-for-more!/id1070071394?mt=8",
+    "linkPlayStore":  "https://play.google.com/store/apps/details?id=com.tredmobile&hl=en",
+    "infoLink": "http://www.geekwire.com/2015/mobile-dealership-tred-raises-another-1m-to-help-used-car-owners-make-more-money/",
+    "infoTitle": "Sell your car for thousands more than Craigslist or the dealer with TRED",
+    "pinned": false
+  },
+  {
+    "name": "Bitt Wallet",
+    "icon": "http://a4.mzstatic.com/us/r30/Purple69/v4/5b/00/34/5b003497-cc85-a0d0-0d3e-4fb3bc6f95cd/icon175x175.jpeg",
+    "linkAppStore": "https://itunes.apple.com/us/app/bitt-wallet/id1081954916?mt=8",
+    "infoLink": "https://bitcoinmagazine.com/articles/overstock-invests-in-bitt-to-launch-official-digital-currencies-in-the-caribbean-islands-1459961581",
+    "infoTitle": "Overstock invests in Bitt to launch digital currencies",
+    "pinned": false
+  },
+  {
+    "name": "CBS Sports Franchise Football",
+    "icon": "http://a2.mzstatic.com/us/r30/Purple69/v4/7b/0c/a0/7b0ca007-885a-7cfc-9fa2-2ec4394c2ecc/icon175x175.png",
+    "linkPlayStore": "https://play.google.com/store/apps/details?id=com.cbssports.fantasy.franchisefootball2015",
+    "infoLink": "http://www.cbssports.com/fantasy/football/games/franchise/2015",
+    "infoTitle": "Award winning Fantasy Football league manager",
+    "pinned": false
```

Comment:
That was my original approach, but I could not find a clean way to filter for unpinned entries in https://github.com/facebook/react-native/pull/12626/files#diff-deb30c36b81d11c5a077a65d4e2d9baaR60 in that case.

Variations I tried that failed to only return objects missing the pinned key (or have pinned: false):

```
const featuredApps = showcaseApps.filter(app => {
  return !app.hasOwnProperty('pinned') || !app.pinned;
}).sort(function(a, b) {
  return a.name.localeCompare(b.name);
});
```